doc/mkdocs.yml

```diff
@@ -8,19 +8,18 @@ copyright: © Red Hat
 nav:
   - Home: index.md
   - Getting Started: getting_started.md
+  - Configuration: configuration.md
   - DBus API: 
     - Description: api/description.md
     - Examples: api/examples.md
-  - Proxy Services: 
-    - Concept: proxy/concept.md
-  - Configuration files:
-    - Hirte controller: man/hirte_conf.md
-    - Hirte agent: man/hirte_agent_conf.md
```

Comment:
I kinda like the configuration files to have their own section, even if that's the man pages indeed.